virttest/shared/cfg/base.cfg

```diff
@@ -945,3 +945,13 @@ nvdimm_uuid = ""
 
 # Update guest default kernel option via serial session
 # kernel_extra_params_serial_login = no
+
+#Set below params to enable cor filter feature.
+#image_copy_on_read: true to make a cor filter insert enabled.
```

Comment:
We can specify the strings to enable the filter, e.g. 
image_copy_on_read: 'yes', 'on' or 'true' to enable the copy-on-read filter

Reply:
Have updated, thanks @zhencliu